packages-openshift.yaml

```diff
@@ -70,3 +80,32 @@ postprocess:
     # Oh right but the MCO overrides that too so...
     mkdir -p /usr/libexec/crio
     ln -sr /usr/bin/conmon /usr/libexec/crio/conmon
+
+  # Inject OpenShift-specific release fields
+  - |
+    cat >> /usr/lib/os-release <<EOF
+    OPENSHIFT_VERSION=4.16
+    EOF
+
+  - |
+     # Let's have a non-boring motd, just like CL (although theirs is more subdued
+     # nowadays compared to early versions with ASCII art).  One thing we do here
+     # is add --- as a "separator"; the idea is that any "dynamic" information should
+     # be below that.
+     # See: https://projects.engineering.redhat.com/browse/COREOS-1029
+     . /etc/os-release
+     colloquial_name=SCOS
+     if [ $ID != "centos" ]; then
+       colloquial_name=RHCOS
```

Comment:
what if this manifest is applied on an FCOS base. Should that be accounted for here?

Reply:
I love the idea of possibly supporting an FCOS base. I think the larger gap there is making sure all the required packages are available and kept up to date, which I think isn't currently the case?

But yeah, for now I think we'll focus on the c9s and RHEL case, but definitely we should think about broadening this to FCOS in the future.